packages/hooks/package.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
I'd suggest a `files` values to reduce bloat of distributed module:

https://docs.npmjs.com/files/package.json#files

e.g. omitting tests.